lib/internal/fs/streams.js

```diff
@@ -155,6 +155,8 @@ ReadStream.prototype._read = function(n) {
     });
   }
 
+  if (this.destroyed) return;
```

Comment:
@ronag I know `_read()`/`_write()` aren’t supposed to be called after `_destroy()`, this is just here because of the delayed calls to these methods from the `once('open', ...)` callbacks … if you happen to be sure that they aren’t necessary let me know and I’ll remove them :)

Reply:
Yep, this is needed until #29656 is merged